client/handle_http_test.go

```diff
@@ -522,11 +560,13 @@ func TestJobIdHeaderSetForDownload(t *testing.T) {
 	assert.NoError(t, err)
 
 	// Write the job ad to the file
-	_, err = jobAdFile.WriteString("GlobalJobId = 12345")
+	_, err = jobAdFile.WriteString("GlobalJobId = \"12345\"")
 	assert.NoError(t, err)
 	jobAdFile.Close()
 
 	os.Setenv("_CONDOR_JOB_AD", jobAdFile.Name())
+	jobAdOnce = sync.Once{}
```

Comment:
There might be a cyclic dependency issue, but can you add a reset of this to one of our test reset functions?

Reply:
Yup -- adds a cyclic dependency.

However, I added a public function to make this available to `cmd` as needed in the future.